opentelekomcloud/services/rds/data_source_opentelekomcloud_rds_instances_v3.go

```diff
@@ -0,0 +1,378 @@
+package rds
```

Comment:
opentelekomcloud/services/rds/data_source_opentelekomcloud_rds_instances_v3.go
should be
opentelekomcloud/services/rds/data_source_opentelekomcloud_rds_instance_v3.go